src/java.base/share/classes/java/util/concurrent/ThreadPoolExecutor.java

```diff
@@ -1512,9 +1507,9 @@ public RejectedExecutionHandler getRejectedExecutionHandler() {
      */
     public void setCorePoolSize(int corePoolSize) {
         if (corePoolSize < 0) {
-            throw new IllegalArgumentException("corePoolSize can't be negative, but got " + corePoolSize);
+            throw new IllegalArgumentException("corePoolSize must be non-negative, but got " + corePoolSize);
         } else if (corePoolSize > maximumPoolSize) {
-            throw new IllegalArgumentException("maximumPoolSize must >= corePoolSize , " +
+            throw new IllegalArgumentException("corePoolSize must be less than or equal to maximumPoolSize, " +
                 "but got maximumPoolSize:" + maximumPoolSize + " corePoolSize :" + corePoolSize);
```

Comment:
```suggestion
                "but got maximumPoolSize: " + maximumPoolSize + ", and corePoolSize : " + corePoolSize);
```

Reply:
I wish we can have a bulit-in IAE formatter for `Preconditions` and then we can call `Preconditions.checkIndex(corePoolSize, maximumPoolSize, Precondtions.IAE_FORMATTER);`